POGOLib.Official/Util/Hash/PokeHashHasher.cs

```diff
@@ -217,10 +216,13 @@ private Task<HttpResponseMessage> PerformRequest(HttpContent requestContent)
                     int maxRequestCount;
                     int rateRequestsRemaining;
                     int ratePeriodEndSeconds;
+                    IEnumerable<string> maxRequestsValue;
+                    IEnumerable<string> requestsRemainingValue;
+                    IEnumerable<string> ratePeriodEndValue;
```

Comment:
It is neccesary, this is the normal use of "out" before of visual studio 2017.
It is a bad use practice for all recomendations of all coder enterprises. Like putting when you want the declaration of variables, instead of the beggining.